deeppavlov/models/embedders/elmo_embedder.py

```diff
@@ -43,24 +43,50 @@ class ELMoEmbedder(Component, metaclass=TfModelMeta):
             ``tenserflow_hub.load_module_spec`` by using `TensorFlow Hub <https://www.tensorflow.org/hub/overview>`__.
         elmo_output_names: A list of output ELMo. You can use combination of
             ``["word_emb", "lstm_outputs1", "lstm_outputs2","elmo"]`` and you can use separately ``["default"]``.
+            where:
+                ``word_emb`` - CNN embedding (default dim 512)
+                ``lstm_outputs*`` - ouputs of lstm (default dim 1024)
+                ``elmo`` - weighted sum of cnn and lstm outputs (default dim 1024)
+                ``default`` - mean ``elmo`` vector for sentence (default dim 1024)
             See `TensorFlow Hub <https://www.tensorflow.org/hub/modules/google/elmo/2>`__ for more information about it.
-        dim: Dimensionality of output token embeddings of ELMo model.
+        dim: Can be used for output embeddings dimensionaliry reduction if elmo_output_names != ['default']
         pad_zero: Whether to use pad samples or not.
         concat_last_axis: A boolean that enables/disables last axis concatenation. It is not used for
             ``elmo_output_names = ["default"]``.
         max_token: The number limitation of words per a batch line.
         mini_batch_size: It is used to reduce the memory requirements of the device.
 
     Examples:
+        >>> from deeppavlov.models.embedders.elmo_embedder import ELMoEmbedder
+        >>> elmo = ELMoEmbedder("http://files.deeppavlov.ai/deeppavlov_data/elmo_ru-news_wmt11-16_1.5M_steps.tar.gz")
+        >>> elmo([['вопрос', 'жизни', 'Вселенной', 'и', 'вообще', 'всего'], ['42']])
+        array([[ 0.00719104,  0.08544601, -0.07179783, ...,  0.10879009,
+                -0.18630421, -0.2189409 ],
+               [ 0.16325025, -0.04736076,  0.12354863, ..., -0.1889013 ,
+                 0.04972512,  0.83029324]], dtype=float32)
+
         You can use ELMo models from DeepPavlov as usual `TensorFlow Hub Module
```

Comment:
Have you checked that this is rendered as expected? (make clean && make html inside docs folder)

Reply:
Checked. Now everything is ok